### PR TITLE
Add two parsers

### PR DESCRIPTION
**Pull request by ChristopherMayes**
This adds special parsers for:
- `Tao.merit`
- `Tao.ele_control_var`

### CODE REVIEW

File: pytao/interface_commands.py [hunk 1]
```diff
@@ -2380,6 +2380,63 @@ def tao_global(tao, *, verbose=False, as_dict=True, raises=True):
...
+     init: -init $ACC_ROOT_DIR/regression_tests/python_test/cesr/tao.init
+     args:
+    
+    """
+    cmd = f'python global:opti_de'
+    if verbose: print(cmd)
+    return __execute(tao, cmd, as_dict, raises, method_name='global_optimation', cmd_type='string_list')
+
+
+def global_opti_de(tao, *, verbose=False, as_dict=True, raises=True):
```
Comment (hhslepicka) [inline pytao/interface_commands.py:2412]: This seems to be the same as above

**Review by hhslepicka — COMMENTED** [review]
There are some weird commands on the optimization side. If they are correct, merge away...

File: pytao/interface_commands.py [hunk 2]
```diff
@@ -2380,6 +2380,63 @@ def tao_global(tao, *, verbose=False, as_dict=True, raises=True):
     return __execute(tao, cmd, as_dict, raises, method_name='tao_global', cmd_type='string_list')
 
 
+def global_optimation(tao, *, verbose=False, as_dict=True, raises=True):
```
[the remark this answers is not in the document]
Comment (ChristopherMayes) [reply pytao/interface_commands.py:2383]: Looks like this is a new command:

```
tao.global_optimation()
{'CR': 0.800000011920929,
 'F': 0.800000011920929,
 'l_best': 0.0,
 'binomial_cross': False,
 'use_2nd_diff': False,
 'randomize_F': False,
 'minimize_merit': True}
```

File: pytao/interface_commands.py (hunk 1 above)
Comment (ChristopherMayes) [reply pytao/interface_commands.py:2412]: Hmm, that's giving the same as above.

Comment (ChristopherMayes) [reply pytao/interface_commands.py:2412]: Okay, I found it. There were typos in `tao/tao_python_cmd.f90`.

**Review by hhslepicka — APPROVED** [review]

**Pull request merged**